style/main.css

```diff
@@ -89,7 +119,7 @@ fieldset {
     -o-transition: background-color 0.3s;
     transition: background-color 0.3s;
 }
-.controlgroup .control:hover, #template-button:hover {
+.control:hover, #template-button:hover {
```

Comment:
Something is giving these buttons a border when hovered over. That makes all the other buttons shift right by 2px when the mouse moves around over them. I can't tell which change gave them this. The buttons have the same set of classes before and after this PR.

Style before:
![image](https://user-images.githubusercontent.com/8954852/54360987-09501e00-4634-11e9-9bbb-94a0b784ea55.png)

Style after:
![image](https://user-images.githubusercontent.com/8954852/54361007-1a992a80-4634-11e9-9fc2-75cdd3e9cb58.png)

Reply:
What we're seeing here is classes at rest. Gotta look at them when `:hover` is active.